src/ucp/core/ucp_context.c

```diff
@@ -1472,7 +1473,7 @@ static ucs_status_t ucp_fill_config(ucp_context_h context,
         goto err_free_alloc_methods;
     }
 
-    context->config.keepalive_interval = ucs_time_from_sec(context->config.ext.keepalive_interval);
+    context->config.keepalive_interval = context->config.ext.keepalive_interval;
```

Comment:
can we remove context->config.keepalive_interval altogether?

Reply:
good catch!
done